src/sentry/templates/sentry/js-sdk-loader.js.tmpl

```diff
@@ -1,31 +1,121 @@
 {% load sentry_helpers %}// Sentry Loader
 (function(_window, _document, _script, _onerror, _onunhandledrejection) {
-  var sentryCallQueue = [];
+  var lazy = 'no';
+  var injected = false;
+  var onLoadCallback;
+
+  // Create a namespace and attach function that will store captured exception
+  // Because functions are also objects, we can attach the queue itself straight to it and save some bytes
+  var queue = function(content) {
+    if (
+      (content.e || content.p ||
+      (content.f && content.f.indexOf('capture') > -1)) &&
+      lazy !== 'no') {
```

Comment:
Can we just use `lazy === 'yes'` instead? Or even better, just `&& lazy` and store it as booleans